locations/spiders/kuveyt_turk_tr.py

```diff
@@ -1,37 +1,78 @@
-from urllib.parse import urlencode, urljoin
+from urllib.parse import urljoin
 
-from scrapy import Spider
 from scrapy.http import JsonRequest, Request
 
-from locations.categories import Categories, apply_category
-from locations.dict_parser import DictParser
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.json_blob_spider import JSONBlobSpider
 
 
-class KuveytTurkTRSpider(Spider):
+class KuveytTurkTRSpider(JSONBlobSpider):
     name = "kuveyt_turk_tr"
     item_attributes = {"brand": "Kuveyt Türk", "brand_wikidata": "Q6036058"}
     allowed_domains = ["kuveytturk.com.tr"]
     base_url = "https://www.kuveytturk.com.tr/"
+    no_refs = True  # Id is not unique across items, it appears to refer to a physical location, so a branch and two ATMs in one place all have the same Id
```

Comment:
I suggest enable deduplication and assign refs depending on type e.g. `item['ref'] = f"{poi.get('id')}-{poi.get('Type')}"`, we'll loose some ATMs, but it should be insignificant since they are inside same building.

Reply:
still, it would be nice to list also these ATMs

Reply:
though if their actual location is not provided, then it would not be so helpful...

Reply:
I think the name was unique maybe that could be used? I'll have a look later

Reply:
> item['ref'] = f"{poi.get('id')}-{poi.get('Type')}"

Dropped a lot of items:

```
 'atp/brand/Kuveyt Türk': 860,
 'atp/brand_wikidata/Q6036058': 860,
 'atp/category/amenity/atm': 412,
 'atp/category/amenity/bank': 448,
```

I'll mash the name into a ref instead, if we want the deduplication pipeline